### ISSUE TITLE
Can't exit and can't close tabs (0.3.1-beta-b34)

### ISSUE THREAD

**Issue by phpbbireland** [kamilfb/mqtt-spy#53]
0.3.1 beta (build 34) on Windows 7

Can't close tabs (right click > unsubscribe and close tab)...
Can't exit the program either... only option is Ctrl+Alt+Del...

Apologies if

**Comment by kamilfb** [comment]
Hi! Any chance you could attach the mqtt-spy.log?

**Comment by phpbbireland** [comment]
Here you go...
```
2015-11-19 10:08:17,979 ERROR [JavaFX Application Thread     ] [SpyUncaughtExceptionHandler   ]  - Thread Thread[JavaFX Application Thread,5,main] failed with java.lang.NoSuchMethodError: pl.baczkowicz.mqttspy.connectivity.MqttAsyncConnection.removeSubscriptionFromMatcher(Lpl/baczkowicz/mqttspy/connectivity/BaseMqttSubscription;)V
java.lang.NoSuchMethodError: pl.baczkowicz.mqttspy.connectivity.MqttAsyncConnection.removeSubscriptionFromMatcher(Lpl/baczkowicz/mqttspy/connectivity/BaseMqttSubscription;)V
	at pl.baczkowicz.mqttspy.connectivity.MqttAsyncConnection.unsubscribe(MqttAsyncConnection.java:322)
	at pl.baczkowicz.mqttspy.connectivity.MqttAsyncConnection.unsubscribeAll(MqttAsyncConnection.java:301)
	at pl.baczkowicz.mqttspy.connectivity.MqttConnectionWithReconnection.disconnect(MqttConnectionWithReconnection.java:93)
	at pl.baczkowicz.mqttspy.ui.connections.ConnectionManager.disconnectFromBroker(ConnectionManager.java:520)
	at pl.baczkowicz.mqttspy.ui.connections.ConnectionManager.disconnectAll(ConnectionManager.java:375)
	at pl.baczkowicz.mqttspy.ui.MainController.exit(MainController.java:391)
	at pl.baczkowicz.mqttspy.ui.MainController$1.handle(MainController.java:164)
	at pl.baczkowicz.mqttspy.ui.MainController$1.handle(MainController.java:1)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(Unknown Source)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(Unknown Source)
	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(Unknown Source)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(Unknown Source)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(Unknown Source)
	at com.sun.javafx.event.EventUtil.fireEventImpl(Unknown Source)
	at com.sun.javafx.event.EventUtil.fireEvent(Unknown Source)
	at javafx.event.Event.fireEvent(Unknown Source)
	at com.sun.javafx.stage.WindowPeerListener.closing(Unknown Source)
	at com.sun.javafx.tk.quantum.GlassWindowEventHandler.run(Unknown Source)
	at com.sun.javafx.tk.quantum.GlassWindowEventHandler.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.tk.quantum.GlassWindowEventHandler.lambda$handleWindowEvent$424(Unknown Source)
	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(Unknown Source)
	at com.sun.javafx.tk.quantum.GlassWindowEventHandler.handleWindowEvent(Unknown Source)
	at com.sun.glass.ui.Window.handleWindowEvent(Unknown Source)
	at com.sun.glass.ui.Window.notifyClose(Unknown Source)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$149(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by kamilfb** [comment]
Thank you for the extra info! I'll look into it tonight and should get a new build by tomorrow.

**Comment by kamilfb** [comment]
Should be fixed in build 38 - please try and confirm if that's any good.

**Comment by kamilfb** [comment]
Fixed in 0.3.1.
